docs/sources/visualizations/histogram-draft.md

```diff
@@ -0,0 +1,107 @@
++++
+title = "Visualize Histograms in Grafana"
+description = "Histogram visualization"
+keywords = ["grafana", "docs", "bar chart", "panel", "prometheus"]
+weight = 605
```

Comment:
Also, update the weight metadata.

Reply:
what should the weight be?

Reply:
For most files, the weight appears to be either 110 or 200 - @achatterjee-grafana - I couldn't find a guideline on this, is there one?